tests/hackage/BUILD

```diff
@@ -14,7 +14,10 @@ load(
 )
 
 [
-    haskell_import(name = name)
+    haskell_import(
+        name = name,
+        deps = ["@gmp"],
```

Comment:
So, is `@gmp` a short-hand for `@gmp//:gmp`? I can't find this documented anywhere.

Reply:
yes apparently